### ISSUE TITLE
how create a executable linux

### ISSUE THREAD

**Issue by rruffer** [lukaszlenart/launch4j-maven-plugin#114]
lauch4j's proposal is to create executables for all platforms. but I'm only able to create windows executables (.exe). How can I configure mavem pom to generate an executable on linux?

note: I started on windows 10 and now I'm developing on ubunto 18. In ubunto it is generating .exe. when i rename the output file without the .exe the lauch4j plug fails.

**Comment by seamanj** [comment]
I have a same problem, could somebody help on this topic?

**Comment by lukaszlenart** [comment]
Nope, as Launch4j states that clearly [here](https://sourceforge.net/projects/launch4j/)
```
Cross-platform Java executable wrapper for creating lightweight Windows native EXEs.
...
The wrapped program works on Windows only, but Launch4j works on Windows, Linux and Mac OS X.
```

So you can build only Windows EXEs, other platforms are not supported. Yet you can build those EXEs on any platform.

**Comment by rruffer** [comment]
What a bad news. Is there any plugin that creates a linux executable. pass parameter in .ini file equal to lauch4j?

**Comment by lukaszlenart** [comment]
Sorry but I don't know :(

**Comment by leewalton** [comment]
@rruffer you could possibly develop a wrapper using kotlin-native that would then spawn a java subprocess to run your application. Kotlin-native is multiplatform, and will build executables for Windows, Mac OSX, Linux, etc

**Comment by rruffer** [comment]
@leewalton java is multiplatform too. But I thought that the only way to start an executable jar passing parameters to vm was with java -jar executable.jar. That's why I got interested in lauch4j. But researching some more I saw that starting a .jar that has a screen the command has to be javaw. In the case java -jar calls console applications and javaw -jar calls applications that have a screen. there it was just doing a .bat and a .sh that everything ran perfectly without the prompt screen.

ex .bat: 
@echo off
start javaw -jar socket-fx-0.0.1.jar
exit
